src/control/Control.Attribution.js

```diff
@@ -19,7 +19,7 @@ export var Attribution = Control.extend({
 	options: {
 		position: 'bottomright',
 
-		// @option prefix: String = 'Leaflet'
+		// @option prefix: String|false = 'Leaflet'

```

Comment:
Technically it looks like any falsy value is will trigger this behavior so `''`, `undefined`, `null` or `0` as well. Is an explicit boolean what we want here?

Reply:
All mentioned values coerce to `false`, so why not make it explicit.